draft-ietf-tls-tls13.md

```diff
@@ -3131,7 +3185,8 @@ aspect of the certificate chain was unacceptable (e.g., it was not signed by a
 known, trusted CA), the server MAY at its discretion either continue the
 handshake (considering the client unauthenticated) or abort the handshake.
 
-Any endpoint receiving any certificate signed using any signature algorithm
+Any endpoint receiving any certificate that must be validated and is
+signed using any signature algorithm
```

Comment:
This is not an editorial nor trivial change. We had consensus for total rejection of MD5 in TLS 1.3+, without the nuance/exceptions/risks we negotiated for SHA1. Something archaic enough to rely on MD5, still, even if it doesn't need to be validated, can be stuck with TLS 1.2 until it gets its act together. This is not to say that some implementation can't ignore the spec and do as you propose (again, there are no IETF police), but we should not be entertaining any legitimate acceptance of MD5 in new specification documents, at this point. If you wish to argue for reopening negotiations on the language here, please propose it on the mailing list with some evidence as to why it might be needed. (If you've got a compelling scenario that can't be fixed otherwise, this is not an unlivable change, but it's not something to stick into a big PR of other fixes/cleanups.)